docs/topics/logging.txt

```diff
@@ -68,9 +68,9 @@ it is passed to a *Handler*.
 Handlers
 ~~~~~~~~
 
-The handler is the engine that determines what happens to each message
-in a logger. It describes a particular logging behavior, such as
-writing a message to the screen, to a file, or to a network socket.
+The *handler* is the engine that determines what happens to each
+message in a logger. It describes a particular logging behavior, such
+as writing a message to the screen, to a file, or to a network socket.
```

Comment:
We wrap docs at 79 chars so you can revert changes in these two lines (there is no need to waste time doing unrelated refactoring of existing code to conform to this style):
```suggestion
The *handler* is the engine that determines what happens to each message
in a logger. It describes a particular logging behavior, such as
writing a message to the screen, to a file, or to a network socket.
```